### PR TITLE
docs(hooks): Document server initialized hooks

### PR DESCRIPTION
**Pull request by vhf**

### CODE REVIEW

File: reference-docs/server-extensions/server-initalization.md [hunk 1]
```diff
@@ -17,3 +17,15 @@ liServer.listen(port, function (err) {
   console.log('Listening on http://localhost:%s', port)
 })
 ```
+
+## Initialized Hooks
+
+Hooks to be executed right after the server initialized can be registered on a Livingdocs Server instance.
```
Comment (peyerluk) [inline reference-docs/server-extensions/server-initalization.md]: We should mention that they are executed before the `listen()` function (before the server starts to listen to a port).

Comment (vhf) [reply reference-docs/server-extensions/server-initalization.md]: Yep, I'll look into it. I'll try to detail the startup steps and where these hooks get executed.

**Comment by vhf** [comment]
@peyerluk I wasn't sure how to explain that the hooks are executed before `listen` since `listen` calls `initialize`.

**Comment by peyerluk** [comment]
@vhf I rewrote it a bit to clarify.

**Comment by peyerluk** [comment]
Merged into: https://github.com/upfrontIO/livingdocs/pull/133

**Pull request closed without merging**